star-gateway/SPI_TRANSPORT_IMPLEMENTATION.md

```
@@ -0,0 +1,604 @@
+- **Gateway Architecture:** `docs/sections/07_gateway_architecture.tex`
+- **HARQ Decision:** `docs/decisions/ADR-001-arq-to-harq.md`
+
+### Source Code
+
+- **Transport:** `internal/transport/spi.go` (276 lines)
+- **Tests:** `internal/transport/spi_test.go` (441 lines)
+- **Main wiring:** `cmd/star-gateway/main.go` (lines 46-78)
+
+### External Resources
+
+- **periph.io docs:** https://periph.io/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)